WORKSPACE

```diff
@@ -19,3 +19,7 @@ envoy_dependencies_extra()
 load("//bazel:dependency_imports.bzl", "envoy_dependency_imports")
 
 envoy_dependency_imports()
+
+load("@bazel_skylib//lib:versions.bzl", "versions")
```

Comment:
@keith was this part necessary ?

if it is im wondering whether it should be in `dependency_imports.bzl` or whether its more explicit in `WORKSPACE`

Reply:
not required, i was just using it as a safety net to make sure that CI was respecting the updated config and using the right bazel version